genapi/src/formula.rs

```diff
@@ -514,7 +500,35 @@ impl<'a> Parser<'a> {
             Expr::Float(f)
         } else {
             let s = self.next_ident().unwrap();
-            Expr::Ident(s)
+            if self.eat(&Token::LParen) {
+                let op = Some(match s.as_str() {
```

Comment:
```suggestion
                let op = match s.as_str() {
```
`op` is no longer needed to have `Option`.

Reply:
Adressed in f419ab06344ba59663868abac1c3ad53c3879ef2